scripts/init-cert/build.sh

```diff
@@ -2,14 +2,21 @@
 
 set -euxo pipefail
 
+TAG=${TAG:-$(git describe --dirty)}
 DKR=${DKR:-docker}
-PUSH_IMAGES=${PUSH_IMAGES:-false}
-
-major=1
-for minor in $(seq 12 18); do
-    tag="k8s-${major}.${minor}"
-    docker build -t elotl/init-cert:${tag} --build-arg K8S_VERSION_MAJOR=${major} --build-arg K8S_VERSION_MINOR=${minor} .
-    $PUSH_IMAGES && docker push elotl/init-cert:${tag} || true
-done
-docker tag elotl/init-cert:k8s-1.18 elotl/init-cert:latest
-$PUSH_IMAGES && docker push elotl/init-cert:latest || true
+REPO=${REPO:-elotl/init-cert}
+PUSH_IMAGES=${PUSH_IMAGES:-true}
+UPDATE_LATEST=${UPDATE_LATEST:-false}
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+cd $SCRIPT_DIR
+
+docker build -t ${REPO}:${TAG} --build-arg K8S_VERSIONS="1.14 1.15 1.16 1.17 1.18" .
```

Comment:
Since updating versions is always a pain and we are already automating the heck out of things, K8S_VERSIONS could also be auto-generated:
```
latest=$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)
latest_minor=$(echo $latest | sed -r 's/v1\.([0-9]+).*/\1/')
k8s_versions=$(seq -f '1.%g' -s ' ' 14 $latest_minor)
echo k8s_versions
```
Not sure if you'd like to add that complexity or whether updating the version string is something that might be a pain to keep track of.  I just got a little jealous of `kubectl.sh` below.

Reply:
Good idea :+1: